rfcs/0120-artifact-service.md

```diff
@@ -242,7 +321,11 @@ local copies of each object.  This cache purging must return a `202 Accepted`
 response and not `200/204` due to the nature of how caches work in the real
 world.
 
-### Retreival
+The `id` parameter in the second endpoint would specify a specific cache id for
+which the deletion is requested.  A special reserved value `all` would request
+that all non-canonical copies of the object be deleted.
```

Comment:
Does this `id` appear elsewhere in the API?  How would a user discover it?

Reply:
My current thinking is that this endpoint would not be used by general users, at least in the first go.  The id would be basically the same as the named origin, but since it's not a request origin, it should have a different name to avoid confusion.